actions/cherry_picker/functions.py

```diff
@@ -144,15 +144,16 @@ def run_cherry_pick(is_prod, commit_id, target_branch_name):
             cherrypick_status = subprocess.run(['git', 'cherry-pick', '-m', '1', commit_id])
 
         lockfile_names = {"src/test/tools/bzlmod/MODULE.bazel.lock", "MODULE.bazel.lock"}
-        unmerged_all_files = str(subprocess.Popen(["git", "diff", "--name-only", "--diff-filter=U"], stdout=subprocess.PIPE).communicate()[0].decode()).split("\n")
-        unmerged_rest = [j for i,j in enumerate(unmerged_all_files) if j not in lockfile_names and j != ""]
+        unmerged_all_files = str(subprocess.Popen(["git", "diff", "--name-only", "--diff-filter=U"], stdout=subprocess.PIPE).communicate()[0].decode())
+        unmerged_rest = [j for i,j in enumerate(unmerged_all_files.split("\n")) if j not in lockfile_names and j != ""]
  
         if cherrypick_status.returncode != 0:
             if len(unmerged_rest) == 0 and ("src/test/tools/bzlmod/MODULE.bazel.lock" in changed_files or "MODULE.bazel.lock" in changed_files):
                 update_lockfile(changed_files, True)
             else:
                 subprocess.run(['git', 'cherry-pick', '--skip'])
-                raise Exception("Cherry-pick was attempted, but there may be merge conflict(s). Please resolve manually.\ncc: @bazelbuild/triage")
+                unmerged_files_names = ("`" + unmerged_all_files.replace("\n", "`\n`"))[:-1]
```

Comment:
we can make this a bit more readable if you keep the `split` on line 147, and use ``"`" + "`\n`".join(unmerged_all_files) + "`"`` here

Reply:
Hi. This works, but I'll have to modify it alittle bit.
```unmerged_files_names = "`" + "`\n`".join(unmerged_all_files[:-1]) + unmerged_all_files[-1] + "`"```

Reply:
is this because the last line is an empty line? if so, you could always `strip()` first on line 147

Reply:
I see. I'll do that